<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tdlist</class>
 <widget class="QWidget" name="tdlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        font-family: &quot;Microsoft YaHei&quot;;
        font-size: 14px;
        color: #333333;
        background-color: #f7f9fb;
    }

    QPushButton {
        background-color: #4CAF50;
        color: white;
        padding: 6px 12px;
        border: none;
        border-radius: 4px;
    }

    QPushButton:hover {
        background-color: #45a049;
    }

    QLineEdit {
        border: 1px solid #ccc;
        padding: 4px;
        border-radius: 4px;
        background-color: #ffffff;
    }

    QLabel {
        color: #2c3e50;
    }

    QTableWidget {
        background-color: white;
        gridline-color: #dcdcdc;
        selection-background-color: #cceeff;
    }

    QHeaderView::section {
        background-color: #e0e0e0;
        padding: 4px;
        border: 1px solid #dcdcdc;
    }</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>任务截止日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>任务名：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="td_del">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tname">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="td_ddl">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tdl_weight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="td_add">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
